src/storage-area.cpp

```diff
@@ -219,3 +219,22 @@ OrthancPluginErrorCode StorageRemoveCallback(const char *uuid, OrthancPluginCont
 
     return OrthancPluginErrorCode_Success;
 }
+
+OrthancPluginErrorCode OnStoredInstanceCallback(const OrthancPluginDicomInstance *instance, const char *instanceId) {
+    char msg[1024] = {0};
+    sprintf(msg, "OnStored Instance Id: %s", instanceId);
+    DEBUG_LOG(1,msg);
+
+    const void* instance_data = OrthancPluginGetInstanceData(globals::context, instance);
+    int size = OrthancPluginGetInstanceSize(globals::context, instance);
+    // update checksum - We need both md5 and size;
+    char* md5 = OrthancPluginComputeMd5(globals::context, instance_data, size);
+
+    memset(msg, 0, sizeof msg);
+    sprintf(msg, "Checksum: MD5 = %s, size = %d", md5, size);
```

Comment:
I think sprintf null terminates

Reply:
It is <https://www.cplusplus.com/reference/cstdio/sprintf/>.
So the memset is superfluous.